package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webgazer",
+  "version": "1.0.0",
+  "description": "Scalable Webcam Eye Tracking Using User Interactions",
+  "main": "index.js",
```

Comment:
"index.js" was the default when running npm init. It isn't kosher, but npm will still install it as a git module.